schematic/models/validate_attribute.py

```diff
@@ -594,9 +758,19 @@ def list_validation(
             list_robustness = "strict"
 
         if list_robustness == "strict":
+            manifest_col = manifest_col.astype(str)
```

Comment:
What if `list_robustness != "strict"` `manifest_col` has no value and line 790 will trigger an error

Reply:
manifest_col is imported, if the list robustness is strict its just converted here to type str for processing within this if.

Reply:
Oh wait yes, disregard this comment :)